tools/picard.py

```diff
@@ -116,20 +119,17 @@ def per_read_group(self, inBam, outDir, picardOptions=None, JVMmemory=None):
                 os.mkdir(outDir)
         else:
             picardOptions = picardOptions or []
-            opts = ['INPUT=' + inBam, 'OUTPUT_DIR=' + outDir, 'OUTPUT_PER_RG=true']
+            opts = ['INPUT=' + inBam, 'OUTPUT_DIR=' + outDir, 'OUTPUT_PER_RG=true', 'RG_TAG=ID']
```

Comment:
Hm, sounds fine to me. Sounds like they wanted to specify whether to break it up by PU or ID (if only they could break it up by LB.. oh well). Tests broke before you added this option, tests fixed after you added it... sure, let's go with that!